apps/management-app/src/store/api/generalManagementApiSlice.ts

```diff
@@ -0,0 +1,269 @@
+import {
+  AdsType,
+  AdsTypeDto,
+  District,
+  DistrictDto,
+  GetListResult,
+  LocationType,
+  LocationTypeDto,
+  MessageResponse,
+  PanelType,
+  PanelTypeDto,
+  ReportType,
+  ReportTypeDto,
+  Ward,
+  WardDto,
+} from '@/types/cdoManagement';
+import { apiSlice } from './baseApiSlice';
+
+export const generalManagementApiSlice = apiSlice.injectEndpoints({
+  endpoints: (build) => ({
+    getDistricts: build.query<
+      GetListResult<District>,
+      { page?: number; limit?: number }
+    >({
+      query: (arg) => ({
+        url: `/districts`,
+        params: {
+          page: arg.page || undefined,
+          limit: arg.limit || undefined,
+        },
+      }),
+    }),
+    getDistrictById: build.query<District, number>({
+      query: (id) => `/districts/${id}`,
+    }),
+    createDistrict: build.mutation<MessageResponse, DistrictDto>({
+      query: (arg) => ({ url: '/districts', method: 'POST', body: arg }),
+    }),
+    updateDistrict: build.mutation<
+      MessageResponse,
+      { id: number; data: DistrictDto }
+    >({
+      query: (arg) => ({
+        url: `/districts/${arg.id}`,
+        method: 'PUT',
+        body: arg.data,
+      }),
+    }),
+    deleteDistricts: build.mutation<MessageResponse, Array<number>>({
+      query: (arg) => ({
+        url: '/districts',
+        method: 'DELETE',
+        body: { ids: arg },
+      }),
+    }),
+    getWards: build.query<
+      GetListResult<Ward>,
+      { page?: number; limit?: number }
+    >({
+      query: (arg) => ({
+        url: `/wards`,
+        params: {
+          page: arg.page || undefined,
+          limit: arg.limit || undefined,
+        },
+      }),
+    }),
+    getWardById: build.query<Ward, number>({
+      query: (id) => `/wards/${id}`,
+    }),
+    createWard: build.mutation<MessageResponse, WardDto>({
+      query: (arg) => ({ url: '/wards', method: 'POST', body: arg }),
+    }),
+    updateWard: build.mutation<MessageResponse, { id: number; data: WardDto }>({
+      query: (arg) => ({
+        url: `/wards/${arg.id}`,
+        method: 'PUT',
+        body: arg.data,
+      }),
+    }),
+    deleteWards: build.mutation<MessageResponse, Array<number>>({
+      query: (arg) => ({
+        url: '/wards',
+        method: 'DELETE',
+        body: { ids: arg },
+      }),
+    }),
+    getPanelTypes: build.query<
+      GetListResult<PanelType>,
+      { page?: number; limit?: number }
+    >({
+      query: (arg) => ({
+        url: `/panelTypes`,
```

Comment:
```suggestion
        url: `/panel-types`,
```
BE also should change like this

Reply:
All endpoints serve as placeholders and will be updated when integrating APIs